config/initializers/assets_paths.rb

```diff
@@ -0,0 +1,5 @@
+%w(stylesheets javascripts fonts images).each do |sub|
+  Rails.application.config.assets.paths << Bootstrap::Rails::Engine.root.join('assets', sub).to_s
+  Rails.application.config.assets.paths << Patternfly::Rails::Engine.root.join('assets', sub).to_s
+  Rails.application.config.assets.paths << FontAwesome::Sass::Rails::Engine.root.join('assets', sub).to_s
+end
```

Comment:
@dLobatog @ohadlevy , This looks to be a decent temp fix for the problem we have.
Looking forward i think we should either do some pull requests for pf,bs,fa sass gems and add a before hook, or change the finisher_hook in our plugins.